bootcamp-milestone-2/src/app/portfolio/page.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+export default function Portfolio() {
+    return (
+        <div>
```

Comment:
You don't actually need this div since everything in the return statement is already wrapped in a <main> tag